.github/workflows/upload-coverage/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Upload Coverage
+description: Upload coverage files
+
+runs:
+  using: "composite"
+
+  steps:
+    - run: |
+        COVERAGE_UUID=$(python3 -c "import uuid; print(uuid.uuid4())")
```

Comment:
shouldn't we use `uuid.uuid4().hex` here? or does `print()` already provide us with that result?

Reply:
So it seems to work well too (without `.hex`).
```bash
(modin) C:\projects\modin>python -c "import uuid; print(uuid.uuid4())     
9990da74-437a-4a2a-94c6-bbd718e5c9b5
```

```bash
(modin) C:\projects\modin>python -c "import uuid; print(uuid.uuid4().hex) 
ef5c0022e478403cae1d9df6dc89719e
```